### ISSUE TITLE
"--mclock 0" not working

### ISSUE THREAD

**Issue by UnPure** [trexminer/T-Rex#494]
I'm using different OC profiles for different algo's and one of them has the best results with memory clock offset of 0. If the memory clock is set to anything else, it does not set the memory to 0 if specified in the start parameters but it seems to work for any other offset. Is this intended?

**Comment by trexminer** [comment]
> it does not set the memory to 0 if specified in the start parameters but it seems to work for any other offset. Is this intended?

Yes, 0 is reserved for "don't touch mem clock" setting, in case you don't want to adjust overclock for one of the cards, e.g.
`--mclock 1000,-502,0,1000`

**Comment by UnPure** [comment]
Is there any way to set it to 0 then? I have 2 different algo's that have the highest hashrate at 0 mclock and would really like to be able to set that somehow without having to use afterburner or other external tools.
